test/framework/executor_itf/executor_itf.hpp

```diff
@@ -61,16 +61,21 @@ namespace iroha {
        * Execute a command as account.
        * @param cmd The command to execute.
        * @param account_id The issuer account id.
+       * @param do_validation Initializes the same parameter of command
```

Comment:
```suggestion
       * @param do_validation /see CommandExecutor
```
or
```suggestion
       * @param do_validation perform permission validation
```

Please check similar doc below.